gatsby_fp/src/pages/404.js

```diff
@@ -1,14 +1,42 @@
 import React from "react"
-
-import Layout from "../components/Layout"
-import SEO from "../components/Seo"
+import Helmet from "react-helmet"
+import Header from "../components/Header"
 
 const NotFoundPage = () => (
-  <Layout>
-    <SEO title="Not Found - Stevens Institute of Technology" />
-    <h1>NOT FOUND</h1>
-    <p>You just hit a route that doesn&#39;t exist... the sadness.</p>
-  </Layout>
+  <>
+    <Header/>
+    <Helmet
+          bodyAttributes={{
+            class: "fs-grid",
+          }}
+    >
+    <title>Not Found - Stevens Institute of Technology</title>
+    </Helmet>
+    <div className="header_push_mobile"></div>
```

Comment:
If there's any way for this tree of markup to exist once instead of twice, we should pursue it. Maybe this file can load the Layout component like it used to, but pass in a prop that tells the component to use "not found" instead of Fields?

Reply:
Hm, since Layout.js is the highest on the prop drilling tree, I do not know if adding props will be feasible because in our gatsby-node.js file, 
our profileTemplate = path.resolve("src/components/Layout.js"), and I do not know if there would be any way to pass props like this.  I think maybe dividing this part of layout.js into it's own component may be easier? @mf-stevens

Reply:
An additional layer of indirection here would probably solve this problem, yes.

Reply:
Took me a while to notice how this works at first, but the GitHub interface has tagged this whole thread "Outdated" since long before your comment, @davidwatson . Basically, the "Conversation" tab shows everything (including Outdated) while the "Files changed" tab shows just those that are outstanding.

Reply:
Oof. Yeah, that's going to take some getting used to.

Thanks Mike!